content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -564,4 +552,22 @@ const product = prisma.product.findFirst({
     }
   },
 })
-``` -->
+
+```
+
+### Order composite types with <inlineCode>orderBy</inlineCode>
+
+You can use the `orderBy` operation in combination with `findMany` to sort results in ascending or descending order.
```

Comment:
Not just findMany, but anywhere `orderBy` is present.

Reply:
I've made ordering into its own subsection, outside of 'Filtering with `find` and `findMany`'